### PR TITLE
Run PHPCS in pipeline

### PR DESCRIPTION
**Pull request by range-of-motion**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -13,5 +13,9 @@ before_script:
...
   - php artisan migrate --database=testing
 
-script:
-  - vendor/bin/phpunit
+jobs:
+  include:
+    - stage: test
+      script: vendor/bin/phpunit
+    - stage: test
+      script: vendor/bin/phpcs app config
```
Comment (sebastiaanspeck) [inline .travis.yml]: You can also specify the folders inside phpcs.xml and when you change that, please add all folders (database, resources (lang, views), routes and tests)

File: phpcs.xml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0"?>
+<ruleset name="Budget">
+  <rule ref="Generic.Arrays.DisallowLongArraySyntax" />
```
Comment (sebastiaanspeck) [inline phpcs.xml]: Extend with more rules (for example with PSR1)

Comment (range-of-motion) [reply phpcs.xml:3]: In due time, let's start

File: .travis.yml (hunk 1 above)
Comment (range-of-motion) [reply .travis.yml:21]: Good point, will do that in the refactor of this PR.

**Pull request merged**